gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Wed Feb 13 09:12:02 CET 2019
```

Comment:
No need to change this file

Reply:
Right, but I have changed it intentionally, since with all distribution gradle source and doc is going to be available from IDEA and you can navigate gradle code and doc from IDEA with the "all" distribution.